routes/year.js

```diff
@@ -115,6 +118,9 @@ router.patch('/api/years/updateYear', auth, async (req, res) => {
     }
     retrievedYear.year = year;
     await retrievedYear.save();
+    await Plans.findByIdAndUpdate(retrievedYear.plan_id, {
+      updatedAt: Date().toLocaleString(),
+    });
```

Comment:
so a plan is updated when a user 
- adds a course 
- deletes a course 
- drags a course 
- update name / year 

what about other updates like update add reviewer or idk fricking add year

Reply:
i think it's honestly good as is! those are the most common updates fosho

Reply:
lol what if we update updatedAt for every GET plans/:plan_id kewk 
would that give us a filter by recently viewed?

Reply:
1. It works for add year(it changes the year array field of the plan object so mongo auto updates updatedAt field). It wouldn't for add/edit reviewer, but wasn't sure if it should(would one reviewer/advisor care about a plan if all a student did was add another advisor to the plan). 
2. Yea updating it for every Get would give us a recently viewed filter(there'd be a little nuance tho with tracking recently updated separately but definitely doable).